www/styles/_base.scss

```diff
@@ -1,100 +1,127 @@
-@use 'tools' as *;
+@use "tools" as *;
```

Comment:
Noticing some link changes in this file. Luckily, we just merged a fix to add stylelint + a pre-commit hook for enforcement. I'd suggest rebasing onto the latest master and giving `npm run lint` a try